### PR TITLE
Update 0000-01-16-alebourne.md

### PR DESCRIPTION
**Pull request by alebourne**
Please review

### CODE REVIEW

**Comment by alebourne** [comment]
@hollenberry please review

**Comment by githubteacher** [comment]
😄

File: _posts/0000-01-16-alebourne.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello all!
```
Comment (githubteacher) [inline _posts/0000-01-16-alebourne.md:6]: Hello!

**Review by githubteacher — APPROVED** [review]
@alebourne this looks great!

**Pull request merged**